boot/modeenv.go

```diff
@@ -273,7 +290,20 @@ func (m *Modeenv) WriteTo(rootdir string) error {
 		}
 		marshalModeenvEntryTo(buf, "model", &modeenvModel{brandID: m.BrandID, model: m.Model})
 	}
+	// TODO: complain when grade or key are unset
 	marshalModeenvEntryTo(buf, "grade", m.Grade)
+	marshalModeenvEntryTo(buf, "sign_key_id", m.SignKeyID)
+	if m.TryModel != "" || m.TryBrandID != "" {
+		if m.Model == "" {
```

Comment:
(nitpick) I wonder if we should add a test for this, but maybe overkill

Reply:
I can add one in a followup